modules/network-observability-flowcollector-configuration-parameters.adoc

```diff
@@ -0,0 +1,98 @@
+* `loki.querierUrl` specifies the address of the Loki query service, in case it is different from the Loki ingester URL. If empty, the URL value will be used (assuming that the Loki ingester and querier are in the same server).
+* `loki.staticLabels` specifies a map of common labels to set on each flow. The default value is `map[app:netobserv-flowcollector]`
+* `loki.statusUrl`specifies the address of the Loki /ready /metrics /config endpoints, in case it is different from the Loki querier URL. If empty, the QuerierURL value will be used. This is useful to show error messages and some context in the frontend.
+* `loki.tenantID` specifies the Loki X-Scope-OrgID that identifies the tenant for each request. It will be ignored if `instanceSpec` is specified. The default value is `netobserv`.
+* `loki.timeout` specifies the maximum time between connection and request. A Timeout of zero means no timeout. The default is `10s` or ten seconds.
+* `loki.tls` allows you to define tls client configurations, such as the following"
+** `caCert` defines the reference of the certificate for the Certificate Authority.
+** `enable` enables TLS. The default is `false`.
+** `insecureSkipVerify` allows skipping client-side verification of the server certificate. If set to true, the `CACert` field is ignored.
+** `userCert`defines the user certificate reference, used for mTLS (you can ignore it when using regular, one-way TLS).
+
+|`spec.namespace`
+| String
+| Specifies the namespace where NetObserv pods are deployed. If empty, the namespace of the operator is going to be used.
+
+|`spec.processor`
+| Object
+a| Specifies the settings of the component that receives the flows from the agent, enriches them, and forwards them to the Loki persistence layer.
+
+* `processor.debug` specifies settings for aspects of the internal configuration of the flow processor. This section is aimed exclusively for debugging and fine-grained performance optimizations, such as GOGC and GOMAXPROCS.
+* `processor.drop.UnusedFields` specifies, when set to true, to drop fields that are known to be unused by OVS, to save storage space.
```

Comment:
typo here, it's `processor.dropUnusedFields`

Reply:
Resolved with generated API